tests/Scripts/license-header.py

```diff
@@ -6,10 +6,8 @@
 for f in sys.argv[1:]:
     with open(f) as input:
         if not any(
```

Comment:
Huh, I'd never seen that before.

Reply:
Already before this was more functional style; the arg to `any` is new a generator expression here. I personally find this still pretty readable, but I could rewrite it e.g., using a temporary list if you could that would help.